Python 2 support in recent releases

### Bug reports

```
___________________________________ ERROR collecting iotlabsshcli/open_a8.py ___________________________________
iotlabsshcli/open_a8.py:25: in <module>
    from iotlabsshcli.sshlib import OpenA8Ssh
iotlabsshcli/sshlib/__init__.py:25: in <module>
    from .open_a8_ssh import OpenA8Ssh
iotlabsshcli/sshlib/open_a8_ssh.py:27: in <module>
    from pssh.clients import SSHClient
.tox/py27-tests/local/lib/python2.7/site-packages/pssh/clients/__init__.py:19: in <module>
    from .native.parallel import ParallelSSHClient
.tox/py27-tests/local/lib/python2.7/site-packages/pssh/clients/native/__init__.py:19: in <module>
    from .parallel import ParallelSSHClient
.tox/py27-tests/local/lib/python2.7/site-packages/pssh/clients/native/parallel.py:20: in <module>
    from .single import SSHClient
.tox/py27-tests/local/lib/python2.7/site-packages/pssh/clients/native/single.py:36: in <module>
    from ..base.single import BaseSSHClient
.tox/py27-tests/local/lib/python2.7/site-packages/pssh/clients/base/single.py:38: in <module>
    from ...output import HostOutput, HostOutputBuffers, BufferData
.tox/py27-tests/local/lib/python2.7/site-packages/pssh/output.py:33: in <module>
    """
E   AttributeError: attribute '__doc__' of 'type' objects is not writable
```

__Expected behaviour:__ 

The python 2 support works in recent releases

__Actual behaviour:__

We wrote a Python library which use Parallel-ssh (thanks for your job). Our library test cases don't work with python 2 due to Parallel-ssh errors. I didn't find in your documentation that you had stopped the Python2 support ? is this the case ?

I just saw that you have circle-ci builds with only Python 3 support. By the way, your documentation (https://parallel-ssh.readthedocs.io/en/latest/) always references old Travis builds.

__Additional info:__ 

parallel-ssh: **2.3.1**
paramiko: **2.7.2**
ssh2-python: **0.25.0**

--- a/Changelog.rst
+++ b/Changelog.rst
@@ -1,6 +1,15 @@
 Change Log
 ============
 
+2.3.2
++++++
+
+Fixes
+-----
+
+* Client output implementation Python 2 support.
+
+
 2.3.1
 +++++
 

--- a/doc/index.rst
+++ b/doc/index.rst
@@ -2,16 +2,14 @@
 Parallel-SSH Documentation
 ===========================
 
-.. image:: https://img.shields.io/badge/License-LGPL%20v2-blue.svg
+.. image:: https://img.shields.io/badge/License-LGPL%20v2.1-blue.svg
   :target: https://pypi.python.org/pypi/parallel-ssh
   :alt: License
 .. image:: https://img.shields.io/pypi/v/parallel-ssh.svg
   :target: https://pypi.python.org/pypi/parallel-ssh
   :alt: Latest Version
-.. image:: https://travis-ci.org/ParallelSSH/parallel-ssh.svg?branch=master
-  :target: https://travis-ci.org/ParallelSSH/parallel-ssh
-.. image:: https://ci.appveyor.com/api/projects/status/github/parallelssh/ssh2-python?svg=true&branch=master
-  :target: https://ci.appveyor.com/project/pkittenis/ssh2-python
+.. image:: https://circleci.com/gh/ParallelSSH/parallel-ssh/tree/master.svg?style=svg
+  :target: https://circleci.com/gh/ParallelSSH/parallel-ssh
 .. image:: https://codecov.io/gh/ParallelSSH/parallel-ssh/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/ParallelSSH/parallel-ssh
 .. image:: https://img.shields.io/pypi/wheel/parallel-ssh.svg

--- a/pssh/output.py
+++ b/pssh/output.py
@@ -18,31 +18,37 @@
 
 """Output module of ParallelSSH"""
 
-from collections import namedtuple
 from os import linesep
 
 from . import logger
 
 
-HostOutputBuffers = namedtuple('HostOutputBuffers', ['stdout', 'stderr'], )
-HostOutputBuffers.__doc__ = """
-:param stdout: Stdout data
-:type stdout: :py:class:`BufferData`
-:param stderr: Stderr data
-:type stderr: :py:class:`BufferData`
-"""
-HostOutputBuffers.stdout.__doc__ = "Stdout :py:class:`BufferData`"
-HostOutputBuffers.stderr.__doc__ = "Stderr :py:class:`BufferData`"
+class HostOutputBuffers(object):
+    __slots__ = ('stdout', 'stderr')
 
-BufferData = namedtuple('BufferData', ['reader', 'rw_buffer'])
-BufferData.__doc__ = """
-:param reader: Reader
-:type reader: :py:class:`gevent.Greenlet`
-:param rw_bufffer: Read/write buffer
-:type rw_buffer: :py:class:`pssh.clients.reader.ConcurrentRWBuffer`
-"""
-BufferData.rw_buffer.__doc__ = "Read/write buffer"
-BufferData.reader.__doc__ = "Greenlet reading data from channel and writing to rw_buffer"
+    def __init__(self, stdout, stderr):
+        """
+        :param stdout: Stdout data
+        :type stdout: :py:class:`BufferData`
+        :param stderr: Stderr data
+        :type stderr: :py:class:`BufferData`
+        """
+        self.stdout = stdout
+        self.stderr = stderr
+
+
+class BufferData(object):
+    __slots__ = ('reader', 'rw_buffer')
+
+    def __init__(self, reader, rw_buffer):
+        """
+        :param reader: Greenlet reading data from channel and writing to rw_buffer
+        :type reader: :py:class:`gevent.Greenlet`
+        :param rw_bufffer: Read/write buffer
+        :type rw_buffer: :py:class:`pssh.clients.reader.ConcurrentRWBuffer`
+        """
+        self.reader = reader
+        self.rw_buffer = rw_buffer
 
 
 class HostOutput(object):

